### PR TITLE
Change dependency and docs for Select

### PR DESCRIPTION
**Pull request by danedavid**
Dependency changed to `rc-select@7.5.0`. Added `onInputKeyDown` to props and docs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "rc-pagination": "~1.12.4",
     "rc-progress": "~2.2.2",
     "rc-rate": "~2.3.0",
-    "rc-select": "~7.3.2",
+    "rc-select": "^7.5.0",
```
Comment (yesmeck) [inline package.json]: Use `~` instead of `^`.

Comment (danedavid) [reply package.json]: still failing

Comment (yesmeck) [reply package.json]: I updated the `feature-3.1` branch, try rebase your branch on it.

**Comment by yesmeck** [comment]
You don't need close this one, after updating your local branch, push your local branch by force to update this PR.

**Comment by danedavid** [comment]
actually it got closed automatically, when I rebased and pushed, without my changes.

**Pull request closed without merging**